### PR TITLE
[devtools-translate] Portuguese - What's New in DevTools (Chrome 104) 

### PR DESCRIPTION
**Pull request by khaosdoctor**
Translate files to PT 

Fix: #3208

### CODE REVIEW

**Comment by khaosdoctor** [comment]
@jecfish done!

**Comment by jecfish** [comment]
wow swift! Thank you! Over to you @alvarocamillont

**Review by andreban — APPROVED** [review]
LGTM, great stuff, thank you!

File: site/pt/blog/new-in-devtools-104/index.md [hunk 1]
```diff
@@ -27,161 +26,210 @@ draft: true
...
 
 <!-- Visualize `performance.measure()` marks in your recording with the new **User Timings** track in the **Performance insights** panel. -->
+Visualize marcas `performance.measure()` em sua gravação com a nova trilha **Tempos do usuário** no painel **Performance insights**.
 
 <!-- For example, this [web page](https://jec.fyi/demo/perf-measure) uses the [`performance.measure()`](https://web.dev/usertiming/#calculating-measurements-with-measure()) method to calculate the elapsed time of text loading. -->
+Por exemplo, esta [página da web](https://jec.fyi/demo/perf-measure) usa o [`performance.measure()`](https://web.dev/usertiming/#calculating-measurements-with-measure()) para calcular o tempo decorrido do carregamento do texto.
 
 <!-- When you start [measuring the page load](/docs/devtools/performance-insights/#record), the **User Timings** track shows in the recording. Click on the timings item to view its details on the side pane. -->
+Quando você começa [medindo o carregamento da página] (/docs/devtools/performance-insights/#record), a trilha **User Timings** é exibida na gravação. Clique no item de tempos para ver seus detalhes no painel lateral.
```
Comment (alvarocamillont) [inline site/pt/blog/new-in-devtools-104/index.md:106]: markdown quebrado [medindo o carregamento da página](/docs/devtools/performance-insights/#record)

**Review by alvarocamillont — COMMENTED** [review]
Some fixes :)

File: site/pt/blog/new-in-devtools-104/index.md [hunk 2]
```diff
@@ -27,161 +26,210 @@ draft: true
...
 
 
 <!-- ## Build an extension for the Recorder panel {: #recorder-extension } -->
+## Crie uma extensão para o painel Recorder {: #recorder-extension}
 
 <!-- You can now build or install a Chrome extension to export replay scripts in your favorite format. See [Recorder extension API](/docs/extensions/reference/devtools_recorder/) documentation to learn how to build one. -->
+Agora você pode criar ou instalar uma extensão do Chrome para exportar scripts de repetição em seu formato favorito. Consulte a documentação da [API de extensão do gravador](/docs/extensions/reference/devtools_recorder/) para saber como criar uma.
 
 <!-- To install a demo extension, follow [these steps](https://github.com/puppeteer/replay#create-a-chrome-extension-for-recorder-available-from-chrome-104-onwards) outlined in the documentation.  -->
+Para instalar uma extensão de demonstração, siga [estas etapas](https://github.com/puppeteer/replay#create-a-chrome-extension-for-recorder-available-from-chrome-104-onwards) descrito na documentação .
```
[the remark this answers is not in the document]
Comment (khaosdoctor) [reply site/pt/blog/new-in-devtools-104/index.md]: Fixed!

File: site/pt/blog/new-in-devtools-104/index.md (hunk 1 above)
Comment (khaosdoctor) [reply site/pt/blog/new-in-devtools-104/index.md]: Fixed

**Comment by khaosdoctor** [comment]
All Fixed :)

**Comment by alvarocamillont** [comment]
Nice! LGTM

**Review by jecfish — APPROVED** [review]

**Comment by jecfish** [comment]
Thank you all!

**Pull request merged**